Common subdirectories: lilo-22.8.ORIG/diagnose and lilo-22.8/diagnose
Common subdirectories: lilo-22.8.ORIG/doc and lilo-22.8/doc
Common subdirectories: lilo-22.8.ORIG/dolilo and lilo-22.8/dolilo
Common subdirectories: lilo-22.8.ORIG/dos and lilo-22.8/dos
diff -u lilo-22.8.ORIG/Makefile lilo-22.8/Makefile
--- lilo-22.8.ORIG/Makefile	2007-03-20 10:29:42.000000000 +0000
+++ lilo-22.8/Makefile	2007-03-20 10:30:07.000000000 +0000
@@ -66,7 +66,7 @@
 
 CONFIG=-DBDATA -DDSECS=3 -DEVMS -DIGNORECASE -DLVM -DNOKEYBOARD -DONE_SHOT \
    -DPASS160 -DREISERFS -DREWRITE_TABLE -DSOLO_CHAIN -DVERSION -DVIRTUAL \
-   -DMDPRAID -DDEVMAPPER
+   -DMDPRAID -DDEVMAPPER -DNOVGA
    
 
 MAN_DIR=/usr/share/man
Common subdirectories: lilo-22.8.ORIG/manPages and lilo-22.8/manPages
Common subdirectories: lilo-22.8.ORIG/sample and lilo-22.8/sample
